journal/vendor-extra/PatternsBundle/Resources/views/patterns/math.html.twig

```diff
@@ -0,0 +1 @@
+<span>{{ math|raw }}</span>{{ '' -}}
```

Comment:
`{{ '' -}}` is just to stop unnecessary whitespace (without removing the newline at the end of the file).